content/intermediate/templates/architecting-templates/index.md

```diff
@@ -0,0 +1,387 @@
+    --template-body file://application-blue-green.template \
+    --parameters \
+        ParameterKey=PageTextContent,ParameterValue=Blue \
+        ParameterKey=RecordSetWeight,ParameterValue=255
+:::
+
+* Create a new stack, called `cloudformation-workshop-dev-application-v2`, to which you pass `Green` as a parameter value for `PageTextContent`, and `1` for `RecordSetWeight`. Example:
+
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws cloudformation create-stack \
+    --stack-name cloudformation-workshop-dev-application-v2 \
+    --template-body file://application-blue-green.template \
+    --parameters \
+        ParameterKey=PageTextContent,ParameterValue=Green \
+        ParameterKey=RecordSetWeight,ParameterValue=1
+:::
+
+* Once both operations are complete, test that the `my-example-domain.com` record mainly points to `Blue`:
+
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+curl http://my-example-domain.com
+:::
+
+* Next, update both stacks, and swap `RecordSetWeight` values:
+
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws cloudformation update-stack \
+    --stack-name cloudformation-workshop-dev-application \
+    --template-body file://application-blue-green.template \
+    --parameters \
+        ParameterKey=PageTextContent,ParameterValue=Blue \
+        ParameterKey=RecordSetWeight,ParameterValue=1
+
+aws cloudformation update-stack \
+    --stack-name cloudformation-workshop-dev-application-v2 \
+    --template-body file://application-blue-green.template \
+    --parameters \
+        ParameterKey=PageTextContent,ParameterValue=Green \
+        ParameterKey=RecordSetWeight,ParameterValue=255
+:::
+
+* Once stack updates are complete, test that the `my-example-domain.com` record now mainly points to `Green`:
+
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+curl http://my-example-domain.com
+:::
+
+::::
+
+
+
+### Clean up
+
+Navigate to the CloudFormation Console on your workstation. Select and delete, in the following order, stacks you created in this lab. *Note that since you used cross-stack references a number of stacks, you cannot delete a stack that exports a value if that value is still used by a consuming stack*:
+
+* Delete `cloudformation-workshop-dev-application-v2` and `cloudformation-workshop-dev-application` without waiting for each one of them has been deleted: such stacks are not depending between each others. When both stacks are deleted, continue with next steps.
+* Delete `cloudformation-workshop-dev-security-groups` and `cloudformation-workshop-dev-hosted-zone` without waiting for each one of them has been deleted as well. Then, continue with next steps.
+* `cloudformation-workshop-dev-cloud9`: this should delete also another stack whose name starts with `aws-cloud9-aws-cloudformation-workshop-`. When both stacks are deleted, continue with the last step shown next.
+* `cloudformation-workshop-dev-base-network`.
+
```

Comment:
```suggestion
1. Delete `cloudformation-workshop-dev-application-v2` and `cloudformation-workshop-dev-application` without waiting for each one of them to be deleted; such stacks do not depend on each other. When both stacks are deleted, continue with next step.
2. Delete `cloudformation-workshop-dev-security-groups` and `cloudformation-workshop-dev-hosted-zone` without waiting for each one of them to be deleted as well. Then, continue with next steps.
3. Delete `cloudformation-workshop-dev-cloud9`: this will delete Cloud9 instance. Also delete stack whose name starts with `aws-cloud9-aws-cloudformation-workshop-`. When both stacks are deleted, continue with the last step.
4. Delete `cloudformation-workshop-dev-base-network` stack.
```

Reply:
As we refer to steps, I suggest changing bullet points to numbered list (I was about to use 1.,1....1, but my IDE auto-corrects that to 1,2,3,4) Feel free to change to 1,1,1,1 :)